public/less/panel_graph.less

```diff
@@ -60,7 +60,6 @@
 }
 
 .graph-legend-table {
-  display: table;
```

Comment:
I had to remove this to make it work with table layout.
All thou this did not change any functionallity so I think it was redundant.